src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+//#region card
+.card {
+  margin-bottom: $vertical-card-ind;
+  background: $color-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $card-width;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+
+  &:not(:nth-child(4n)) {
+    margin-right: $horizontal-card-ind;
+  }
+
+  &:nth-child(4n) {
+    margin-bottom: $vertical-card-ind;
+  }
+
+  &:not(:nth-last-child(-n +4)) {
+    margin-bottom: 46px;
+  }
+
+  &__logo {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__description {
+    font-family: inherit;
+    display: block;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-color-accent;
+    margin: 0 $card-min-margin-side 4px;
+  }
+
+  &__code {
+    text-align: left;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin-bottom: 16px;
+    margin-left: $card-min-margin-side;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 $card-min-margin-side 24px;
+  }
+
+  &__prices {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 $card-min-margin-side 16px;
+    box-sizing: border-box;
+  }
+
+  &__text {
+    color: #616070;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__amount {
+    color: $main-color-accent;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__link-buy {
+    background-color: $color-accent-blue;
+    border-radius: 5px;
+    border: 1px solid transparent;
+    margin: 0 $card-min-margin-side 16px;
+    box-sizing: border-box;
+    display: flex;
```

Comment:
try to organize you styles 
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)